plookup/src/multiset.rs

```diff
@@ -337,7 +337,7 @@ mod test {
         let alpha = BlsScalar::from(2);
         let alpha_squared = alpha * alpha;
 
-        let mut table = WitnessTable3Arity::new();
+        let mut table = WitnessTable3Arity::default();
```

Comment:
We should use `min_const_generics` for this WitnessTable's with different arity values known beforehand.

Could you file an issue for it?